extensions/adobe/experience/adcloud/searchads/campaign.example.1.json

```diff
@@ -0,0 +1,14 @@
+{
+  "_adcloud.searchads:id": "100001061",
+  "_adcloud.searchads:campaignId": "10001234",
```

Comment:
This should be "searchads:campaignId": "10001234". 
Use the exact field name from the XDM schema here.

Reply:
Fixed example for all fields